product_subscription/__openerp__.py

```diff
@@ -1,41 +1,28 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Business Open Source Solution
-#    Copyright (C) 2013-2016 Coop IT Easy SCRL.
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2019 Coop IT Easy SCRL fs
+#   Houssine Bakkali <houssine@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
 {
-    "name": "Product Subscription",
-    "version": "1.0",
-    "depends": [
-        "sale",
-        "partner_firstname",
-        "account_analytic_distribution",
-        "l10n_be_invoice_bba",
-        "email_template_config",
-        "mail",
+    'name': 'Product Subscription',
+    'version': '1.0',
+    'depends': [
+        'account',
+        'sale',
+        'stock',
+        'partner_firstname',
+        'account_analytic_distribution',
+        'l10n_be_invoice_bba',  # todo delocalize
+        'email_template_config',
+        'mail',
     ],
-    "author": "Houssine BAKKALI <houssine@coopiteasy.be>",
-    "category": "Sales",
-    "website": "www.coopiteasy.be",
-    "description": """
+    'author': 'Houssine BAKKALI <houssine@coopiteasy.be>',
```

Comment:
we didn't decided to put coop it easy as author ?

Reply:
:+1: